releases/note-source/v20_0_0.rst

```diff
@@ -38,33 +38,176 @@ For a complete list of changes made, see :ref:`release-v20-0-0-tickets`.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`fgcmcal <lsst.fgcmcal>` tool can now correct for the local background on aperture fluxes during calibration.
+This brings faint stars more into line with their bright counterparts, and thereby improves the resultant calibration.
+:jirab:`DM-23036`
+
+.. _release-v20-0-0-gen2to3:
+
+Conversion of Generation 2 to Generation 3 data repositories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is now possible to convert Generation 2 to Generation 3 data repositories using the :command:`butler convert` command.
+For details, refer to the :ref:`obs_base <lsst.obs.base>` :ref:`lsst.obs.base-cli`.
+:jirab:`DM-22655`
 
 .. _release-v20-0-0-interface:
 
 Significant Interface Changes
 -----------------------------
 
+- :ref:`release-v20-0-0-conda`
+- :ref:`release-v20-0-0-fits-headers`
+- :ref:`release-v20-0-0-collections`
+- :ref:`release-v20-0-0-lsstsw-envconfig`
+- :ref:`release-v20-0-0-jointcal-gaia`
 - :ref:`release-v20-0-0-remove-obs_lsstSim`
 - :ref:`release-v20-0-0-remove-afwGeom`
 - :ref:`release-v20-0-0-remove-PixelScaleBoundedField`
 - :ref:`release-v20-0-0-remove-addDistortionModel`
 - :ref:`release-v20-0-0-remove-_extractAmpId`
-- :ref:`release-v20-0-0-remove-pex_policy`
 - :ref:`release-v20-0-0-remove-astrometry_net_data`
 - :ref:`release-v20-0-0-remove-getHasRawInfo`
 - :ref:`release-v20-0-0-remove-pixelScale`
-- :ref:`release-v20-0-0-remove-obs_sdss`
 
+.. _release-v20-0-0-conda:
+
+Third party packages managed through Conda
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `Conda <https://conda.io>`_ package management system, together with the `Conda-Forge <https://conda-forge.org>`_ package collection, is now used to provide all compilers and most third-party dependencies.
+This substantially simplifies :ref:`part-installation` and makes it easier to integrate the Science Pipelines with other tools, while also facilitating quicker updates to third-party code during development.
+
+For more information, refer to `community.lsst.org <https://community.lsst.org/t/conda-updates-implementing-the-switch-to-conda-forge-conda-compilers-rfc-679/4127>`_.
+:jirab:`RFC-679, DM-24232`
+
+.. _release-v20-0-0-fits-headers:
+
+All FITS headers are now stored in uppercase
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The version of `CFITSIO`_ used in this release requires that all header keys be written in uppercase to better comply with the `FITS standard`_.
+Before this release, some Science Pipelines code would store information in FITS headers in lowercase.
+This has the consequence that:
+
+- If user code attempts to write a lowercase header to FITS, it will be automatically converted to uppercase, and a warning will be issued;
+- If user code attempts to read FITS data which includes lowercase headers (for example, because it was written by an earlier Pipelines release), they will automatically be converted to uppercase, and a warning will be issued.
```

Comment:
```suggestion
- If user code attempts to read FITS data which includes lowercase headers (for example, because it was written by an earlier Pipelines release), they will automatically be converted to uppercase, and an informational message may be logged.
```